schemas/app-sre/schedule-1.yml

```
@@ -2,40 +2,65 @@
 "$schema": /metaschema-1.json
 version: "1.0"
 type: object
+title: "Schedule"
+description: |
+  Schema for defining a schedule used to automate Slack user group assignments.
```

I'd mention this config is not related to slack but mostly used for slack user group assignment. We could reuse this schema for something else than slack.
```suggestion
  Schema for defining a schedule used to automate user group assignments. This is most notably used for Slack user groups via a permission file.
```

---

Why not update the description when the use case changes, instead of implying a future use case that is currently unsupported?